### PR TITLE
Jw pricing map bug

### PR DESCRIPTION
**Pull request by jwiel86**
Diffed failing specs with failing specs on master

### CODE REVIEW

File: app/assets/javascripts/catalog_manager/update.js.coffee [hunk 1]
```diff
@@ -96,6 +96,34 @@ $(document).ready ->
   $('.service_rate').live('blur', ->
     $(this).formatCurrency()
   )
+
```
Comment (jayhardee9) [inline app/assets/javascripts/catalog_manager/update.js.coffee]: You can attach the same listener to multiple elements using commas like:
`$('[display="Federal Rate"], [display="Corporate Rate"], [display="Other Rate"], etc.').live(...)`

File: spec/features/catalog_manager/create_pricing_map_spec.rb [hunk 2]
```diff
@@ -53,8 +54,9 @@
 
       fill_in "pricing_maps_blank_pricing_map_full_rate", with: 4321
       fill_in "otf_quantity_type_", with: "hours"
-
+      sos
```
Comment (jayhardee9) [inline spec/features/catalog_manager/create_pricing_map_spec.rb]: Two screenshots!

Comment (jwiel86) [inline spec/features/catalog_manager/create_pricing_map_spec.rb]: Woopsies!

File: app/assets/javascripts/catalog_manager/update.js.coffee (hunk 1 above)
Comment (jwiel86) [inline app/assets/javascripts/catalog_manager/update.js.coffee]: Cool! Thanks!

**Comment by wtholt** [comment]
Looks good to me

**Pull request merged**